Tests/BenchmarkTests/BenchmarkReporterTests.swift

```diff
@@ -41,13 +41,19 @@ final class BenchmarkReporterTests: XCTestCase {
     }
 
     func assertReported(_ got: String, _ expected: String) {
+        func trimmingTrailingWhitespace(_ string: String) -> String {
+            String(string.reversed().drop(while: { $0.isWhitespace }).reversed())
+        }
+        
         let lines = Array(got.split(separator: "\n").map { String($0) })
         let expectedLines = expected.split(separator: "\n").map { String($0) }
         let actual = lines.map { $0.trimmingCharacters(in: .newlines) }
             .filter { !$0.isEmpty }
         XCTAssertEqual(expectedLines.count, actual.count)
         for (expectedLine, actualLine) in zip(expectedLines, actual) {
-            XCTAssertEqual(expectedLine, actualLine)
+            let trimmedExpectedLine = trimmingTrailingWhitespace(expectedLine)
+            let trimmedActualLine = trimmingTrailingWhitespace(actualLine)
```

Comment:
The benchmark printer produces trailing whitespace (and the current set of tests simply happened to not include any).

Including trailing whitespace in test files however is prone to breakage due to IDEs commonly stripping it on save, so we instead strip both, `actual` and `expected`, before comparing them.